src/lib/theme/themeUtils.ts

```
@@ -1,7 +1,19 @@
 
 export function getTheme<K extends keyof ThemeConfig>(componentKey: K) {
   const theme = getContext<ThemeConfig>("theme");
   return theme?.[componentKey];
 }
+
+type Slots<T extends { slots: any }> = Omit<T["slots"], "base">
+export type Classes<T extends { slots: Record<string, any> }> = {
+  classes?: Partial<{ [K in keyof Slots<T>]: ClassValue; }>;
+};
+
+export function themeDeprecated(component: string, names: { [x: string]: any }): void {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)